src/Link.js

```diff
@@ -1,7 +1,7 @@
 import styled from '@emotion/styled'
 import theme from './theme'
 
-let LinkButton = styled.button({
+let Link = styled.span({
```

Comment:
I think this is really a Button of type `link` - it has much more in common with a button than an anchor because it doesn't support `href`

Reply:
we don't use this component much (I almost killed it), but most of our usage is actually inline links, for which `span` is substantially better:

<img width="1017" alt="Screen Shot 2020-02-11 at 2 26 57 PM" src="https://user-images.githubusercontent.com/35466670/74271277-96837300-4cda-11ea-9835-4c3efbee37d0.png">

(I mean, we could also default it to a button and do `as="span"` if we need it inlined, but that seems backwards imo)

the only real advantage of using `button` in this case is that it's more semantic, but we don't generally care about that anyway

Reply:
`create-react-app` specifically suggests using buttons when you use links without hrefs which is where this suggestion came from. We don't have to make it an actual `Button`, we shouldn't call it `Link` because most uses of the identifier `Link` would be for the navigation kind (e.g., `react-router5`'s `Link`)

Reply:
renamed to LinkText